git-link.el

```diff
@@ -328,6 +334,9 @@ return (FILENAME . REVISION) otherwise nil."
   (and (boundp 'git-timemachine-revision)
        git-timemachine-revision))
 
+(defun git-link--using-magit-blame ()
+  magit-blame-mode)
```

Comment:
This will result in an error if `magit-blame-mode` is not defined. You'll have to do something like [` git-link--using-git-timemachine`](https://github.com/sshaw/git-link/blob/1dbabfed4c5c3c5ac6ffa9035a9c3d4c6cc7f885/git-link.el#L327-L329).